packages/app/src/runs/RunsContainer.vue

```diff
@@ -27,23 +27,39 @@
       data-cy="runs"
       class="flex flex-col pb-[24px] gap-[16px]"
     >
-      <Warning
+      <TrackedBanner
         v-if="userProjectStatusStore.cloudStatusMatches('needsRecordedRun') && userProjectStatusStore.project.isUsingGit"
         :title="t('runs.empty.noRunsFoundForBranch')"
-        :message="noRunsForBranchMessage"
-      />
+        :banner-id="ACI_052023_NO_RUNS_FOUND_FOR_BRANCH"
+        dismissible
+        status="warning"
+        :has-banner-been-shown="false"
+        :event-data="undefined"
+      >
+        <div
+          ref="markdownTarget"
+          class="warning-markdown"
+          v-html="markdown"
+        />
+      </TrackedBanner>
       <Warning
         v-if="!online"
         :title="t('launchpadErrors.noInternet.header')"
         :message="t('launchpadErrors.noInternet.message')"
         :dismissible="false"
         class="mx-auto mb-[24px]"
       />
-      <Warning
+      <TrackedBanner
```

Comment:
Nice re-use of this existing banner, forgot we had the `TrackedBanner` abstraction.